resources/assets/js/components/pages/home/NewSpot.vue

```diff
@@ -62,8 +62,9 @@
                                 value=""
                                 size="small"
                                 class="full-width"
-                                v-if="descriptor.value_type === 'select'"
+                                v-if="descriptor.value_type.toLowerCase().includes('select')"
```

Comment:
Select and MultiSellect descriptors share almost identical markup, this detects both cases of value types.